doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -558,3 +558,1274 @@ each region.
 +-------------+---------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
 | eu-west-1   | ``613661167059.dkr.ecr.eu-west-1.amazonaws.com/amazon-sagemaker-operator-for-k8s:v1``       | https://amazon-sagemaker-operator-for-k8s-eu-west-1.s3.amazonaws.com/kubectl-smlogs-plugin/v1/linux.amd64.tar.gz       |
 +-------------+---------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+
+
+Using Amazon Sagemaker Jobs
+---------------------------
+
+To run a job using the Amazon Sagemaker Operators for Kubernetes, you can either apply
+a YAML file or use the supplied Helm charts.
+
+All operator sample jobs in the following tutorials use sample data
+taken from a public MNIST dataset. In order to run these samples, download the dataset into your S3 bucket. You can find
+the dataset in \ `Download the MNIST
+Dataset. <https://docs.aws.amazon.com/sagemaker/latest/dg/ex1-preprocess-data-pull-data.html>`__
+
+.. contents::
+
+TrainingJob operator
+~~~~~~~~~~~~~~~~~~~~
+
+Training job operators reconcile your specified training job spec to
+Amazon SageMaker by launching it for you in Amazon SageMaker. You can
+learn more about Amazon SageMaker training jobs in the Amazon
+SageMaker \ `CreateTrainingJob API
+documentation <https://docs.aws.amazon.com/sagemaker/latest/dg/API_CreateTrainingJob.html>`__.
+
+Create a TrainingJob Using a Simple YAML File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Download the sample YAML file for training using the following command: 
```

Comment:
This feels like content that would be better suited for an example notebook. rather than writing a tutorial here, can you just touch on the highlights about how to work with the operators? the reader shouldn't have to download files as part of consuming this content.

here's an example of one of our other doc pages: https://sagemaker.readthedocs.io/en/stable/amazon_sagemaker_processing.html

and if you have suggestions on how to better explain what kind of content lives in this part of the documentation, let me know so we can update the [guidelines for writing documentation in CONTRIBUTING.md](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#documentation-guidelines)